.changeset/dirty-tables-speak.md

```diff
@@ -0,0 +1,224 @@
+---
+'@chainlink/instruxi-sxt-adapter': major
```

Comment:
Why change so many EA

Reply:
I assume this was unintentional. @PatricioMacedo  only the one line for `@chainlink/instruxi-sxt-adapter` will be necessary

Reply:
Yes, apologies this was unintentional. Should I make a new PR with all the changes?

Reply:
You can push a new revision here

Reply:
I have pushed, and sent env to Sam

Reply:
There is still many many EAs here, can you delete all the files under .changeset and run yarn changeset again?

Reply:
Ok,deleted,  ran it again and pushed